src/app/components/text-image-component/text-image-component.component.css

```diff
@@ -0,0 +1,38 @@
+.container {
+  width: 90%;
+  height: max-content;
+  overflow: hidden;
+  margin: auto;
+  position: relative;
+}
+
+.text-container {
+  width: max-content;
+  max-width: 50%;
+  height: auto;
+  font-size: large;
+  overflow: hidden;
+  text-align: center;
+  margin-top: 100px
+}
+
+.container img {
+  width: 50%;
+  height: auto;
+}
+
+.left img {
+  float: left;
+}
+
+.left div {
+  float: right;
+}
+
+.right img {
+  float: right;
+}
+
+.right div {
+  float: left;
+}
```

Comment:
Make the image smaller, change the color of the text and add the styling for the title.
Also don't make our component as wide as the screen.